### PR TITLE
Sentence correction for Info section on concepts/architecture/nodes

### PR DESCRIPTION
**Pull request by manishbansal8843**
Sentence corrected

### CODE REVIEW

**Comment by manishbansal8843** [comment]
/check-cla
Forcing cla check again

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -111,9 +111,9 @@ on a Node.
 
 ### Info
 
-General information about the node, such as kernel version, Kubernetes version
-(kubelet and kube-proxy version), Docker version (if used), OS name.
-The information is gathered by Kubelet from the node.
+Describes the general information about the node, such as kernel version, Kubernetes version
```
Comment (zacharysarah) [inline content/en/docs/concepts/architecture/nodes.md]: Replace with:
> Describes general information about the node, such as kernel version and Kubernetes version.

File: content/en/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -111,9 +111,9 @@ on a Node.
...
+(kubelet and kube-proxy version), Docker version (if used) and OS name.
```
Comment (zacharysarah) [inline content/en/docs/concepts/architecture/nodes.md]: Replace with:
> (kubelet and kube-proxy version), Docker version (if used), and OS name.

File: content/en/docs/concepts/architecture/nodes.md (hunk 1 above)
Comment (manishbansal8843) [reply content/en/docs/concepts/architecture/nodes.md]: I think `full stop` is not required after `Kubernetes version` as it is the same line.

**Pull request merged**